js_parser/README.md

```diff
@@ -37,23 +33,15 @@ laptop.  jsparagus is slow.
 Once you're done, to see your parser run, try this:
 
 ```console
-$ python -m js_parser.try_it
-```
-
-
-### Or the Rust version
-
-Run all the steps above, except substitute this command for the one that ends in `.py`:
-
-```console
-$ python -m js_parser.generate_js_parser_tables js_parser/parser_generated.jsparagus_dump -o rust/parser/src/parser_generated.rs
+$ cd rust
+$ cargo run --release
 ```
 
-Then, to see your parser run, try this:
+The build also produces a copy of the JS parser in Python.
+After `make all`, you can use this command to run it:
 
 ```console
-$ cd rust/parser
-$ cargo run --release
+$ jsparagus_build_venv/bin/python -m js_parser.try_it
```

Comment:
hm that doesn't look so great. Is there a point that we might reach where we would want this to be cleaner? Will we be adding a lot more python?

Reply:
Hmm. Well, we could tell people to activate the venv:

```
$ source jsparagus_build_env/bin/activate
$ python -m js_parser.try_it
```

Better?

Reply:
Never mind, there's a target in the Makefile for this, although its name `make jsdemo` is not ideal.